index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <header id="header">
+        <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"></img>
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#section1">Features</a></li>
+            <li><a class="nav-link"  href="#section2">How It Works</a></li>
+            <li><a class="nav-link"  href="#section3">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+
+
+
+      <section id="hero">
+        <h2>Handcrafted, home-made masterpieces</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+          <input id="submit" type="submit" value="Get Started" class="btn">
+        </form>
+     </section>
+     <table>
+
+      <tr>
```

Comment:
The tag section and table are different tags, please make a new line space between these tags. 👍 
I think your code would more readable if you remove the blank spaces between table and tr tags 👍